pkg/apis/core/v1beta1/defaults_shoot_test.go

```diff
@@ -288,6 +288,33 @@ var _ = Describe("Shoot defaulting", func() {
 
 				Expect(obj.Spec.Kubernetes.Kubelet.SerializeImagePulls).To(PointTo(BeFalse()))
 			})
+
+			It("should not overwrite already set values for maxParallelImagePulls field", func() {
+				var (
+					limitParallelImagePulls int32 = 5
+				)
+
+				obj.Spec.Kubernetes.Kubelet = &KubeletConfig{}
+				obj.Spec.Kubernetes.Kubelet.MaxParallelImagePulls = &limitParallelImagePulls
+
+				SetObjectDefaults_Shoot(obj)
+
+				Expect(obj.Spec.Kubernetes.Kubelet.MaxParallelImagePulls).To(PointTo(Equal(int32(5))))
+			})
+
+			It("should set serializeImagePulls to false when maxParallelImagePulls field is set", func() {
```

Comment:
```suggestion
			It("should default serializeImagePulls to false when maxParallelImagePulls field is set", func() {
```
Also I thing it makes sense `It("should default the serializeImagePulls field"` to be changed to:
`should default serializeImagePulls to true when maxParallelImagePulls is unset or set to less than 2`